multichain-testing/tools/agd-tools.ts

```diff
@@ -8,7 +8,7 @@ export const makeAgdTools = async (
     execFileSync,
   }: Pick<typeof import('child_process'), 'execFile' | 'execFileSync'>,
 ) => {
-  const bundleCache = unsafeMakeBundleCache('bundles');
+  const bundleCache = await unsafeMakeBundleCache('bundles');
```

Comment:
how was this discovered? do we need more CI for multichain-testing?

maybe that's why you're proposing it be in force-integration